CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Deprecate `setup_future_usage` from `checkoutComplete.paymentData` input - will be removed in Saleor 4.0 - #7994 by @mateuszgrzyb
 - Possibility to pass metadata in input of `checkoutPaymentCreate` - #8076 by @mateuszgrzyb
 - Fix shipping address issue in `availableCollectionPoints` resolver for checkout - #8143 by @kuchichan
+- Do not create gift cards for waiting for approval orders - #8162 by @IKarbowiak
```

Comment:
I'm not sure we have to update changelog in this case - this is part of the gift cards feature that will be introduced in 3.1 and we don't need to describe every change. I will just include this in release notes but I will know about this PR from the list of merged PRs.

Reply:
Ok. I'll remove that.